tests/testthat/test-readwrite.R

```diff
@@ -83,3 +83,10 @@ test_that("empty files lead to NULL or error", {
     SS_readstarter("nonexistentfile.ss", verbose = FALSE)
   ))
 })
+
+test_that("ss_read works with a URL", {
+  skip_if_offline(host = "github.com")
+  list_objs <- SS_read(dir = 
+  "https://github.com/nmfs-stock-synthesis/test-models/tree/main/models/Simple")
```

Comment:
You have to use raw user content. See the example in `SS_read()`. We should probably add something to the documentation about how to call from GitHub? 🤷

Reply:
Ah, of course. I added a little bit to the documentation to hopefully guide the user to use the raw content!